packages/nft-checkout/README.md

```diff
@@ -33,48 +48,53 @@ const sendTransaction = async () => {
 
   // Select the chain to pay from.
   // This example uses the Goerli chain, but your application can ask the user which chain to use.
-  const selectedChain = chains.find(i => i.name === ChainNames.Goerli)
-
-  // Set the parameters for the transaction, including the price and the tokens to accept payment in.
-  const target = {
-    // Destination chain id (Sepolia in this case)
-    chainId: 11155111,
-    // Recipient's wallet address
-    recipient: "0x0000000000000000000000000000000000000000",
-    // The amount of token to swap the payment token to
-    price: Price.fromRaw("0.01", 18, 'ETH'),
+  const selectedChain = chains.find((i: BridgeChain) => i.name === selectedChainName)!
+
+  // Select the chain to pay on.
+  // In this case, the NFT contract is on the Sepolia chain.
+  const destinationChain = chains.find((i: BridgeChain) => i.name === targetChainName)!
+
+  // Set the parameters for the transaction, including source and destination chain.
+  const params:CheckoutOperationParams = {
+    chainIdFrom: selectedChain.id,
+    chainIdTo: destinationChain.id,
+    price: Price.fromRaw("0.01", 18, "ETH"),
   }
 
-  // Initialize the operation with the source chain and transaction parameters.
-  await op.init({
-    chainIdFrom: selectedChain!.id,
-    target,
-  })
+  // Initialize the transaction object
+  await op.init(params)
 
   // Load the user's balance of payment tokens on the source chain.
   // When this method runs, the wallet prompts the user to switch to the selected chain if necessary.
   // Then, the method returns the tokens on this chain that the DEX supports and that the wallet has a balance of greater than zero.
-  const tokens = await op.loadPaymentTokens(selectedChain!)
+  const paymentTokens = await op.loadPaymentTokens(selectedChain)
 
   // Select the token to accept payment in on the source chain.
   // This example hard-codes UNI, but your application can ask the user which token to pay with.
-  const paymentToken = tokens[0]
+  const selectedToken = paymentTokens?.find(i => i.symbol === paymentToken)!
 
   // Get the estimated purchase price in the payment token, including the cost to swap the tokens to the tokens that the seller accepts payment in.
   // At this point you can ask the user to confirm the transaction with the fees or cancel it.
-  const estimatedPrice = await op.estimatePrice(paymentToken)
+  const estimatedPrice = await op.estimatePrice(selectedToken)
+
+  // Create a JSON string with pricing information to encode in the transaction.
+  const priceString = JSON.stringify({
+    impact: estimatedPrice.impact,
+    price: estimatedPrice.price.toString(),
+    gasPrice: estimatedPrice.gasPrice,
+  })
```

Comment:
This info is only for the user, and u don't even use this anywhere

Reply:
Yep, no longer using that in the bundle. Removed.